robottelo/cli/contentview.py

```diff
@@ -35,14 +35,256 @@
+                                                    'updated'
+            --end-date END_DATE                     erratum: end date
+                                                    (YYYY-MM-DD)
+            --errata-id ERRATA_ID                   erratum: id
+            --errata-ids ERRATA_IDS                 erratum: IDs or a select
+                                                    all object
+                                                    Comma separated list of
+                                                    values.
+            --max-version MAX_VERSION               package: maximum version
+            --min-version MIN_VERSION               package: minimum version
+            --name NAME                             package and package group
+                                                    names
+                                                    Comma separated list of
+                                                    values.
+            --names NAMES                           Package and package group
+                                                    names
+            --start-date START_DATE                 erratum: start date
+                                                    (YYYY-MM-DD)
+            --types TYPES                           erratum: types
+                                                    (enhancement, bugfix,
+                                                    security)
+                                                    Comma separated list of
+                                                    values.
+            --version VERSION                       package: version
+
+        """
+        if (
+                not options or
+                'content-view-filter' not in options and
+                'content-view-filter-id' not in options):
+            raise CLIError(
+                'Could not find content_view_filter, please set one of options'
+                ' "content-view-filter" or "content-view-filter-id".'
+            )
+        cls.command_sub = 'create'
+        result = cls.execute(
+            cls._construct_command(options), output_format='csv')
+
+        # Extract new CV filter rule ID if it was successfully created
+        if len(result) > 0 and 'id' in result[0]:
+            cvfr_id = result[0]['id']
+            # CV filter rule can only be fetched by specifying either
+            # content-view-filter-id or content-view-filter + content-view-id.
+            # Passing these options to info command
+            info_options = {
+                'content-view-id': options.get('content-view-id'),
+                'content-view-filter': options.get('content-view-filter'),
+                'content-view-filter-id': options.get(
+                    'content-view-filter-id'),
+                'id': cvfr_id,
+            }
+            result = cls.info(info_options)
+        return result
+
+    @classmethod
+    def update(cls, options=None):
```

Comment:
Just information for user as there's no method body, just call for super. This is how most (?) of cli modules are implemented and i like this style - if you're not sure which parameters are accepted you can simply go to method definition in your IDE and see that.